jsonargparse/_parameter_resolvers.py

```diff
@@ -898,6 +923,23 @@ def get_field_data_attrs(field, name, doc_params):
     )
 
 
+def is_init_field_pydantic_model(field) -> bool:
+    return True
```

Comment:
This is somewhat weird. Why not just a `lambda f: True` where this is used? Then it is clearer there that it is always supported.